.gitignore

```diff
@@ -250,4 +250,21 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
-# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/python,pycharm
+
+
+# IDEs
+.idea
+.vscode
+.settings
+.project
```

Comment:
What is using `.project` and `.settings` ?

Reply:
eclipse

Reply:
Then use this one: https://www.toptal.com/developers/gitignore/api/eclipse
![image](https://user-images.githubusercontent.com/7374733/135086061-485b6bb1-3f8c-4597-b372-f4bd43cb8e0d.png)